bin/duo-install

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
```

Comment:
Done! I've added `lib/utils.js` which is a collection of helper functions used by both `_duo` and `duo-install`.